### DISCUSSION TITLE
Implementing Orca LLM by Microsoft in Open Assistant if they somehow release it

### DISCUSSION THREAD

**Discussion by arsg0etia** [LAION-AI/Open-Assistant#3340] [Ideas]
I recently learned about Orca, a new open-source LLM developed by Microsoft Research. It is built on llama13b and has been shown to outperform ChatGPT on the BBH benchmark and perform at par with text-davinci-003 in the AGIEval reasoning benchmark. Orca is a 13B model that has been trained on large-scale training data with diverse tasks augmented with complex instructions and rich signals.

I think it would be interesting to explore the possibility of implementing Orca LLM in Open Assistant. Since Orca is an open-source model, it should be possible to integrate it into Open Assistant as an additional language model option. This would require some development work to ensure that Orca can be accessed and used through Open Assistant's API or platform. It would also be important to ensure that Orca's performance is properly benchmarked and tested within the context of Open Assistant.

Overall, implementing Orca LLM in Open Assistant could potentially provide users with a powerful new language model option that has been shown to perform well on various benchmarks. This could enhance the capabilities of Open Assistant and provide users with even more accurate and high-quality AI performance. However, it would require some development work to properly integrate Orca into Open Assistant's existing framework.

What do you think about this idea? Is this something that the Open Assistant team would be interested in exploring further?

**Comment by gururise** [comment]
Where can one download the orca weights? We will need the weights before any integration can occur.

**Reply by yatesdr** [reply]
The weights aren’t released yet, but MS committed to releasing the delta weights as open source.   No date announced yet that I’ve been able to find.
